testing/spectest/shared/deneb/operations/withdrawals.go

```diff
@@ -0,0 +1,49 @@
+package operations
+
+import (
+	"context"
+	"path"
+	"testing"
+
+	"github.com/golang/snappy"
+	"github.com/prysmaticlabs/prysm/v4/beacon-chain/core/blocks"
+	"github.com/prysmaticlabs/prysm/v4/beacon-chain/state"
+	consensusblocks "github.com/prysmaticlabs/prysm/v4/consensus-types/blocks"
+	"github.com/prysmaticlabs/prysm/v4/consensus-types/interfaces"
+	enginev1 "github.com/prysmaticlabs/prysm/v4/proto/engine/v1"
+	ethpb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v4/testing/require"
+	"github.com/prysmaticlabs/prysm/v4/testing/spectest/utils"
+	"github.com/prysmaticlabs/prysm/v4/testing/util"
+)
+
+func RunWithdrawalsTest(t *testing.T, config string) {
+	require.NoError(t, utils.SetConfig(t, config))
+	testFolders, testsFolderPath := utils.TestFolders(t, config, "deneb", "operations/withdrawals/pyspec_tests")
+	for _, folder := range testFolders {
+		t.Run(folder.Name(), func(t *testing.T) {
+			folderPath := path.Join(testsFolderPath, folder.Name())
+			payloadFile, err := util.BazelFileBytes(folderPath, "execution_payload.ssz_snappy")
+			require.NoError(t, err)
+			payloadSSZ, err := snappy.Decode(nil /* dst */, payloadFile)
+			require.NoError(t, err, "Failed to decompress")
+			payload := &enginev1.ExecutionPayloadDeneb{}
+			require.NoError(t, payload.UnmarshalSSZ(payloadSSZ), "Failed to unmarshal")
+
+			body := &ethpb.BeaconBlockBodyDeneb{ExecutionPayload: payload}
+			RunBlockOperationTest(t, folderPath, body, func(_ context.Context, s state.BeaconState, b interfaces.SignedBeaconBlock) (state.BeaconState, error) {
+				payload, err := b.Block().Body().Execution()
+				if err != nil {
+					return nil, err
+				}
+				withdrawals, err := payload.Withdrawals()
+				if err != nil {
+					return nil, err
+				}
+				p, err := consensusblocks.WrappedExecutionPayloadCapella(&enginev1.ExecutionPayloadCapella{Withdrawals: withdrawals}, 0)
```

Comment:
wait shouldn't this type be deneb?

Reply:
fixed in 9f2d53754588ce54b10e297e8c765246964e069c